### PR TITLE
DF-1457: Save user Language

### PR DESCRIPTION
**Pull request by pavax**
…fOfWorkEfforts and ApplicationDocument

### CODE REVIEW

File: domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package ch.admin.seco.alv.service.onlineform.domain;
+
+public enum Language {
+	DE,
+	FR,
+	IT,
+	EN
```
Comment (dvoracek-martin) [inline domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java]: English is never used and throughout the whole system, we work with DE, FR, IT only. When it's some other value, it goes to the default value DE

Comment (emina-str) [reply domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java]: @pavax  could we delete EN language?

Comment (pavax) [reply domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java]: yes

File: domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/bu/ApplicationDocument.java [hunk 2]
```diff
@@ -67,12 +63,14 @@ private ApplicationDocument() {
 		this.ownerUserId = Condition.notNull(builder.ownerUserId);
 		this.documentMetadata = new ApplicationDocumentMetadata(builder.document, builder.fileName);
 		this.personNumber = Condition.notNull(builder.personNumber);
+		this.language = Condition.notNull(builder.language);
 		this.createdAt = TimeMachine.now();
 		this.updatedAt = this.createdAt;
 	}
 
-	public void updateDocumentType(ApplicationDocumentType newDocumentType) {
+	public void updateDocumentTypeAndLanguage(ApplicationDocumentType newDocumentType, Language language) {
```
Comment (pavax) [inline domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/bu/ApplicationDocument.java]: revert

File: domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java (hunk 1 above)
Comment (emina-str) [reply domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/Language.java]: removed

File: domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/bu/ApplicationDocument.java (hunk 2 above)
Comment (emina-str) [reply domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/bu/ApplicationDocument.java]: renamed

**Pull request merged**